src/test/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusinessIntegrationSpec.scala

```diff
@@ -18,48 +16,43 @@ class AgoraBusinessIntegrationSpec extends FlatSpec with BeforeAndAfterAll with
+    val actual = patiently(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get))
+    assert(actual == entity)
   }
 
   ignore should "be unable to store a task configuration with an invalid docker image (invalid/non-existent repo name)" in {
-    val thrown = intercept[DockerImageNotFoundException] {
-      agoraBusiness.insert(testAgoraEntityWithInvalidOfficialDockerRepoNameInWdl, mockAuthenticatedOwner.get)
+    intercept[DockerImageNotFoundException] {
+      patiently(agoraBusiness.insert(testAgoraEntityWithInvalidOfficialDockerRepoNameInWdl, mockAuthenticatedOwner.get))
     }
-    assert(thrown != null)
   }
 
   ignore should "be unable to store a task configuration with an invalid docker image (invalid/non-existent tag name)" in {
-    val thrown = intercept[DockerImageNotFoundException] {
-      agoraBusiness.insert(testAgoraEntityWithInvalidOfficialDockerTagNameInWdl, mockAuthenticatedOwner.get)
+    intercept[DockerImageNotFoundException] {
+      patiently(agoraBusiness.insert(testAgoraEntityWithInvalidOfficialDockerTagNameInWdl, mockAuthenticatedOwner.get))
     }
-    assert(thrown != null)
   }
 
   "Agora" should "be able to store a task configuration with a valid (extant) personal docker image" in {
-    val entity = agoraBusiness.insert(testAgoraEntityWithValidPersonalDockerInWdl, mockAuthenticatedOwner.get).addManagers(Seq(mockAuthenticatedOwner.get))
-    assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get) === entity)
+    val entity = patiently(agoraBusiness.insert(testAgoraEntityWithValidPersonalDockerInWdl, mockAuthenticatedOwner.get)).addManagers(Seq(mockAuthenticatedOwner.get))
+    assert(patiently(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), mockAuthenticatedOwner.get)) == entity)
   }
 
   ignore should "be unable to store a task configuration with an invalid personal docker image (invalid/non-existent user name)" in {
-    val thrown = intercept[DockerImageNotFoundException] {
-      agoraBusiness.insert(testAgoraEntityWithInvalidPersonalDockerUserNameInWdl, mockAuthenticatedOwner.get)
+    intercept[DockerImageNotFoundException] {
+      patiently(agoraBusiness.insert(testAgoraEntityWithInvalidPersonalDockerUserNameInWdl, mockAuthenticatedOwner.get))
     }
-    assert(thrown != null)
   }
 
   ignore should "be unable to store a task configuration with an invalid personal docker image (invalid/non-existent repo name)" in {
-    val thrown = intercept[DockerImageNotFoundException] {
-      agoraBusiness.insert(testAgoraEntityWithInvalidPersonalDockerRepoNameInWdl, mockAuthenticatedOwner.get)
+    intercept[DockerImageNotFoundException] {
+      patiently(agoraBusiness.insert(testAgoraEntityWithInvalidPersonalDockerRepoNameInWdl, mockAuthenticatedOwner.get))
     }
-    assert(thrown != null)
```

Comment:
Nice. There are a few more of these that I noticed in EntityPermissionsClientSpec if you're interested in more clean-up.